test/spec/modules/sovrnBidAdapter_spec.js

```diff
@@ -307,206 +277,251 @@ describe('sovrnBidAdapter', function() {
-        expect(payload.site.domain).to.equal('example.com')
+          }
+          return utils.deepAccess(cfg, key)
+        })
+
+        const request = spec.buildRequests([baseBidRequest], baseBidderRequest)
+        const { user, site } = JSON.parse(request.data)
+
+        expect(user.data).to.equal('some user data')
+        expect(site.keywords).to.equal('test keyword')
+        expect(site.page).to.equal('http://example.com/page.html')
+        expect(site.domain).to.equal('example.com')
       })
       it('should append impression first party data', function () {
-        const fpdBid = {...adUnitBidRequest}
-        fpdBid.ortb2Imp = {
-          ext: {
-            data: {
-              pbadslot: 'homepage-top-rect',
-              adUnitSpecificAttribute: '123'
+        const fpdBidRequest = {
+          ...baseBidRequest,
+          ortb2Imp: {
+            ext: {
+              data: {
+                pbadslot: 'homepage-top-rect',
+                adUnitSpecificAttribute: '123'
+              }
             }
           }
         }
-        const request = spec.buildRequests([fpdBid], bidderRequest)
+
+        const request = spec.buildRequests([fpdBidRequest], baseBidderRequest)
         const payload = JSON.parse(request.data)
+
         expect(payload.imp[0].ext.data.pbadslot).to.equal('homepage-top-rect')
         expect(payload.imp[0].ext.data.adUnitSpecificAttribute).to.equal('123')
       })
       it('should not overwrite deals when impression fpd is present', function() {
-        const fpdBid = {...adUnitBidRequest}
-        fpdBid.params = {...adUnitBidRequest.params}
-        fpdBid.params.segments = 'seg1, seg2'
-        fpdBid.ortb2Imp = {
-          ext: {
-            data: {
-              pbadslot: 'homepage-top-rect',
-              adUnitSpecificAttribute: '123'
+        const fpdBid = {
+          ...baseBidRequest,
+          params: {
+            segments: 'seg1, seg2'
+          },
+          ortb2Imp: {
+            ext: {
+              data: {
+                pbadslot: 'homepage-top-rect',
+                adUnitSpecificAttribute: '123'
+              }
             }
           }
         }
-        const request = spec.buildRequests([fpdBid], bidderRequest)
-        const payload = JSON.parse(request.data)
-        expect(payload.imp[0].ext.data.pbadslot).to.equal('homepage-top-rect')
-        expect(payload.imp[0].ext.data.adUnitSpecificAttribute).to.equal('123')
-        expect(payload.imp[0].ext.deals).to.deep.equal(['seg1', 'seg2'])
+
+        const request = spec.buildRequests([fpdBid], baseBidderRequest)
+        const impression = JSON.parse(request.data).imp[0]
+
+        expect(impression.ext.data.pbadslot).to.equal('homepage-top-rect')
+        expect(impression.ext.data.adUnitSpecificAttribute).to.equal('123')
+        expect(impression.ext.deals).to.deep.equal(['seg1', 'seg2'])
       })
     })
-  });
+  })
 
   describe('interpretResponse', function () {
```

Comment:
I am not seeing the unittest for above code.

Reply:
I add unit test for new functionality [here](https://github.com/sovrn/Prebid.js/pull/34/commits/8b09f9148473c9c108a7ea6b5aeed3f95a880ddd#diff-9b3cbd20715f7d1fc68496aca92c279070d2a0bc1d5cc004c47792272d1e39adR461)

Reply:
I have updated files. Now tests (here)[https://github.com/sovrn/Prebid.js/pull/34/files#diff-9b3cbd20715f7d1fc68496aca92c279070d2a0bc1d5cc004c47792272d1e39adR474]